pkg/chunkenc/pool.go

```diff
@@ -376,9 +376,6 @@ type BufioReaderPool struct {
 // Get returns a bufio.Reader which reads from r. The buffer size is that of the pool.
 func (bufPool *BufioReaderPool) Get(r io.Reader) *bufio.Reader {
 	buf := bufPool.pool.Get().(*bufio.Reader)
```

Comment:
Q: is it ok to panic if this type inference fails?

Reply:
it shouldn't panic.